app/(tabs)/card/details.tsx

```diff
@@ -0,0 +1,47 @@
+import { Image } from "expo-image";
+import React from "react";
+import { View } from "react-native";
+
+import { Button } from "@/components/ui/button";
+import { IconSymbol } from "@/components/ui/IconSymbol";
+import { Text } from "@/components/ui/text";
+
+export default function CardDetails() {
+  // Mock card data - in a real app this would come from an API or secure storage
+  const cardData = {
+    balance: "$1,234.56",
+    spendableBalance: "$1,234.56",
+    status: "Active"
+  };
+
+  return (
+    <View className="flex-1 bg-background p-6">
+      {/* Balance Information */}
+      <View className="items-center mb-6">
+        <View className="flex-row items-baseline">
+          <Text className="text-[18px] font-semibold">$</Text>
```

Comment:
@SametSahin10 avoid using arbitrary className `[]`, and `px` for text. Use `text-lg`